library/rental.rb

```diff
@@ -0,0 +1,27 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.rentals << self
+
+    @book = book
+    book.rentals << self
+  end
+
+  def to_s
+    "Date: #{@date}, Book \"#{book.title}\" by #{book.author}"
+  end
+
+  def to_json(*args)
+    {
+      JSON.create_id => self.class.name,
+      'date' => @date,
+      'book_title' => @book.title,
+      'person_id' => @person.id
+    }.to_json(*args)
+  end
```

Comment:
- [ ] After creating a rental, when I am trying to fetch the rental, the application crashes, and also it does not preserve data. Please fix that.